rbac/managedclustersets/hcm-cicd-int/rolebinding.yaml

```diff
@@ -0,0 +1,15 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: hcm-cicd-int-managedclusterset-admin
+subjects:
+  - kind: Group
+    apiGroup: rbac.authorization.k8s.io
+    name: hcm-cicd-int
```

Comment:
You'll want to set this value to `hcm-delivery` because that's your Kube RBAC Group!